File: Collect.jl
using CSV, DataFrames, PrettyTables

df = CSV.read("data_with_classes.csv", DataFrame)

df = groupby(df, :Class)

matclass1 = df[1]
matclass1 = select(matclass1, Not(:Class))
matclass1 = Matrix(matclass1)

matclass2 = df[2]
matclass2 = select(matclass2, Not(:Class))
matclass2 = Matrix(matclass2)

matclass3 = df[3]
matclass3 = select(matclass3, Not(:Class))
matclass3 = Matrix(matclass3)

pretty_table(matclass1)
println(size(matclass1))
pretty_table(matclass2)
println(size(matclass2))
pretty_table(matclass3)
println(size(matclass3))

File: Collect.py
import pandas as pd

df = pd.read_csv('data_with_classes.csv')
# print(df)

matclass1 = df[df['Class'] == 1]
del matclass1['Class']
matclass1 = matclass1.values
# matclass1 = matclass1.transpose()

matclass2 = df[df['Class'] == 2]
del matclass2['Class']
matclass2 = matclass2.values
# matclass2 = matclass2.transpose()

matclass3 = df[df['Class'] == 3]
del matclass3['Class']
matclass3 = matclass3.values
# matclass3 = matclass3.transpose()

print(matclass1)
print(matclass1.shape)
print()
print(matclass2)
print(matclass2.shape)
print()
print(matclass3)
print(matclass3.shape)